gcp/modules/gke-cluster/main.tf

```diff
@@ -19,10 +19,11 @@ module "gke_cluster" {
 
   initial_node_count = 1
   node_type          = "${var.node_type}"
-  kubernetes_version = "1.10.9-gke.5"
 
-  main_compute_zone = "us-central1-a"
-  additional_zones  = ["us-central1-b", "us-central1-c", "us-central1-f"]
+  kubernetes_version = "1.11.3-gke.18"
+
+  region           = "us-central1"
+  additional_zones = ["us-central1-a", "us-central1-b", "us-central1-c", "us-central1-f"]
```

Comment:
I think this is going to increase number of worker nodes per cluster, do we want that?

Reply:
No, because we removed the `zone = "us-central1-a"` and changed the cluster to regional, now the list of zones in `additional_zones` is all the zones you get... the `additional` part in the name is hence quite confusing, I agree. The cluster size is still going to be `4`.

Reply:
Ok, makes sense, thanks!